app/src/enclave_u/mod.rs

```diff
@@ -76,15 +75,15 @@ pub fn check_initchain(eid: sgx_enclave_id_t, chain_hex_id: u8) -> Result<(), ()
 pub fn check_transfertx(
     eid: sgx_enclave_id_t,
     txaux: TxAux,
-    txins: Vec<TxWithOutputs>,
+    txins: Vec<Vec<u8>>,
```

Comment:
about about 
type Vec<u8>= Tx
Vec<Tx>

Reply:
Not possible -- `app/src/enclave_u` = untrusted code which cannot unseal the data (as it's not in the enclave)